### PR TITLE
Update topLevelFrames.ts

### PR DESCRIPTION
**Pull request by Andreslav**
Added ability to pass pageNode.

### CODE REVIEW

**Comment by Andreslav** [comment]
@thomas-lowry, look please.

File: src/helpers/getPage.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //this function allows you to pass in a node and return its pageNode
-export default function getPage(node: BaseNode): PageNode {
+export default function getPage(node: BaseNode) {
```
Comment (alexandrtovmach) [inline src/helpers/getPage.ts]:
```suggestion
export default function getPage(node: BaseNode): node as PageNode {
```

File: src/helpers/getPage.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 	while (node && node.type != 'PAGE') {
 		node = node.parent
 	}
-	return node
+	return node as PageNode
```
Comment (alexandrtovmach) [inline src/helpers/getPage.ts]:
```suggestion
	return node
```

File: src/helpers/getPage.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 //this function allows you to pass in a node and return its pageNode
-export default function getPage(node: BaseNode): PageNode {
+export default function getPage(node: BaseNode) {
 	while (node && node.type != 'PAGE') {
```
Comment (alexandrtovmach) [inline src/helpers/getPage.ts]:
```suggestion
	while (node && node.type !== 'PAGE') {
```

File: src/helpers/getPage.ts (hunk 1 above)
Comment (Andreslav) [reply src/helpers/getPage.ts]: Not valid..

File: src/helpers/getPage.ts (hunk 3 above)
Comment (Andreslav) [reply src/helpers/getPage.ts]: @alexandrtovmach Thank you, corrected

File: src/helpers/getPage.ts (hunk 1 above)
Comment (alexandrtovmach) [reply src/helpers/getPage.ts]: sorry, it should be `node is PageNode` instead of `node as PageNode`

**Pull request merged**